tests/logic.js

```diff
@@ -171,7 +171,7 @@ define([
                 var form = util.loadXML(MOTHER_REF_XML),
                     manager = form._logicManager;
                 assert.deepEqual(manager.caseReferences(),
-                    {load: {"/data/mug": ["parent/edd"]}});
+                    {load: {"/data/mug": ["#case/parent/edd"]}});
```

Comment:
Sorry I didn't say that very well.

This is an issue with the Case Summary in HQ . The reason we remove the #case was to provide parity with old case management and how it saves case properties. Currently the Case Summary expects that form which has always been `property` or `parent/property`.

`caseReferences` is a method to provide HQ with loaded case properties to display in the Case Summary (https://github.com/dimagi/Vellum/pull/832/files#diff-e8c10ecfceaf2d2a578b7321207927c9R463)

If we send back `#case/dob` instead of `dob` we'll end up with something like 
![image](https://cloud.githubusercontent.com/assets/1471773/21197306/ffdee5e2-c208-11e6-8a73-4e7c7f05d589.png)

In the future it could also show `#user/property` if we don't filter that, or come up with a different way to store these things.

I'm pro sending `#case/property`  and `#user/property`, so that these can be differentiated, but it will also require changes to the case summary in HQ. That could possibly be just removing `#case` when displaying in the summary, but there is probably a better way. The summary still needs to support the old form of `property` or `parent/property` for displaying saved case properties